### PR TITLE
add task solution moyo header

### PR DESCRIPTION
**Pull request by EdwardYatos**
- [DEMO LINK] https://EdwardYatos.github.io/layout_moyo-header/
- [TEST REPORT LINK] https://EdwardYatos.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,99 @@
...
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+ul {
```
Comment (mykhalenych) [inline src/style.css:17]:
```suggestion
```
delete it

File: src/style.css [hunk 2]
```diff
@@ -1 +1,99 @@
...
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (mykhalenych) [inline src/style.css:48]:
```suggestion
  align-items: center;
  padding: 0;
  margin: 0;
  list-style-type: none;
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,99 @@
...
+.nav-link {
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  text-transform: uppercase;
+  line-height: 14px;
+}
+
+.nav-link:hover,
+a.is-active {
```
Comment (mykhalenych) [inline src/style.css:80]: you already have class for link use instead of tag name

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 4]
```diff
@@ -1 +1,102 @@
...
+.nav-items {
+  margin-right: 20px;
+  padding: 23px 0;
+}
+
+.nav-items:last-child {
+  margin-right: 0;
+}
+
+.nav-items a {
```
Comment (YegorKorenkov) [inline src/style.css:63]: please use class selector instead of `a`

File: src/index.html [hunk 5]
```diff
@@ -1,14 +1,43 @@
...
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
```
Comment (YegorKorenkov) [inline src/index.html:18]: this container redundant, it doesn't do anything, please delete

File: src/style.css [hunk 6]
```diff
@@ -1 +1,102 @@
...
+  height: 60px;
+  margin: 0 auto;
+}
+
+.main-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
```
Comment (YegorKorenkov) [inline src/style.css:32]: Tru to not hardcode height

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Looks like your code and your demo linkd not consistent
try to deploy your changes
Screenshot from 2023-05-13 16-37-29

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**